### PR TITLE
[next] Docker + Revision Metadata

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

- Adds Docker release steps to the CircleCI config.
- Adds revision information, and exposes it via the `/api/version` route.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,11 +29,12 @@ jobs:
           command: npm audit
       - run:
           name: Install dependencies
-          command: npm install
+          command: npm ci
       - save_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
           paths:
             - ./node_modules
+            - ~/.npm
```
Comment (alinhle) [inline .circleci/config.yml:41]: What made you include this?

File: .circleci/config.yml [hunk 2]
```diff
@@ -113,3 +126,11 @@ workflows:
       - build:
           requires:
             - npm_dependencies
+      - release:
```
Comment (alinhle) [inline .circleci/config.yml]: Shouldn't this be `release_docker`?

File: .circleci/config.yml (hunk 1 above)
Comment (wyattjoh) [reply .circleci/config.yml:41]: https://docs.npmjs.com/cli/ci

File: .circleci/config.yml (hunk 2 above)
Comment (wyattjoh) [reply .circleci/config.yml]: ... Yes

**Review by alinhle — APPROVED** [review]

**Pull request merged**